packages/components/organisms/f-cookie-banner/test/component/f-cookieConsentBanner.component.shared.spec.js

```diff
@@ -26,10 +31,15 @@ describe('New - f-cookieBanner component tests - @browserstack', () => {
     });
 
     // 'dk' and 'no' disabled for now
-    forEach(['es', 'ie', 'it'])
+    forEach(['es-ES', 'en-IE', 'it-IT'])
     .it('should display the f-cookieBanner component for "%s"', tenant => {
         // Arrange
-        cookieBanner.open(tenant);
+        cookieBanner = new CookieBanner('organism', 'cookie-banner-component');
```

Comment:
Do we need this `cookieBanner` when we're already calling it in the top level describe?

Reply:
ahh yeah we do - I've just checked and it errors if I delete the new instance (just line 37) sadly

Reply:
Ok cool. I've seen a couple instances of this in other tests and wanted to check if this was right. All good then. :)

Reply:
It needs a new instance to override the old one that's in memory from another test.

Reply:
else you get conflicts with URL arguments that have different values (knobs)